Packs/Smokescreen_IllusionBLACK/Integrations/Smokescreen_IllusionBLACK/Smokescreen_IllusionBLACK.yml

```diff
@@ -95,6 +103,10 @@ script:
     description: Check if a subdomain is a Threat Intel decoy
     execution: false
     name: illusionblack-is-subdomain-decoy
+    outputs:
+    - contextPath: IllusionBlack.IsSubdomainDecoy
+      description: A mapping of subdomain and whether it's an IllusionBLACK TI decoy
+      type: Unknown
```

Comment:
```suggestion
      type: Boolean
```

Reply:
This is not boolean, it is a dict:
```
{
  "HOST": true/false
}
```

Reply:
Sorry, me bad, then the contextPath should point out to `IllusionBlack.IsSubdomainDecoy.HOST`

Reply:
Added context output:

- IllusionBlack.IsHostDecoy
- IllusionBlack.IsHostDecoy.Host
- IllusionBlack.IsHostDecoy.Value

```
{
    "IllusionBlack": {
        "IsHostDecoy": {
            "Host": "SAP44",
            "Value": true
        }
    }
}
```